grails-app/views/cycleCount/_printRecount.gsp

```diff
@@ -0,0 +1,147 @@
+            </div>
+        </div>
+
+        <div class="content">
+            <g:each var="cycleCount" in="${cycleCounts}">
+                <g:set var="product" value="${cycleCount.cycleCountItems?.first()?.product}"/>
+                <g:set var="dateCounted" value="${cycleCount.cycleCountItems?.first()?.dateCounted}"/>
+                <g:set var="userCounted" value="${cycleCount.cycleCountItems?.first()?.assignee}"/>
+                <table>
+                    <thead>
+                    <tr>
+                        <th colspan="6" class="b-r0 b-b0" style="padding: 10px 0 10px 10px;">
+                            ${warehouse.message(code: 'cycleCount.product.label')}: ${product?.name}
+                        </th>
+                        <th align="right" class="b-l0 b-b0" style="padding: 10px 10px 10px 0;">
+                            ${warehouse.message(code: 'cycleCount.productCode.label')}: ${product?.productCode}
+                        </th>
+                    </tr>
+                    <tr>
+                        <th colspan="2" class="b-r0 b-t0 b-b0" style="padding: 0 0 10px 10px;">
+                            ${warehouse.message(code: 'cycleCount.dateCounted.label')}:
+                            <g:formatDate date="${dateCounted}" format="MMM d, yyyy"/>
+                        </th>
+                        <th colspan="2" class="b-r0 b-l0 b-t0 b-b0" style="padding: 0 0 10px 0;">
+                            ${warehouse.message(code: 'cycleCount.userCounted.label')}:
+                            ${userCounted}
+                        </th>
+                        <th colspan="3" class="b-l0 b-t0 b-b0" style="padding-bottom: 10px;"></th>
+                    </tr>
+                    <tr>
+                        <th colspan="2" class="b-r0 b-t0" style="padding: 0 0 10px 10px;">
+                            ${warehouse.message(code: 'cycleCount.dateRecounted.label')}:
+                        </th>
+                        <th colspan="2" class="b-r0 b-l0 b-t0" style="padding: 0 0 10px 0;">
+                            ${warehouse.message(code: 'cycleCount.userRecounted.label')}:
+                        </th>
+                        <th colspan="3" class="b-l0 b-t0" style="padding-bottom: 10px;"></th>
+                    </tr>
+                    <tr>
+                        <th>${warehouse.message(code: 'cycleCount.binLocation.label')}</th>
+                        <th>${warehouse.message(code: 'cycleCount.lotNumber.label')}</th>
+                        <th>${warehouse.message(code: 'cycleCount.expirationDate.label')}</th>
+                        <th>${warehouse.message(code: 'cycleCount.quantityCounted.label')}</th>
+                        <th>${warehouse.message(code: 'cycleCount.difference.label')}</th>
+                        <th>${warehouse.message(code: 'cycleCount.quantityRecounted.label')}</th>
+                        <th>${warehouse.message(code: 'cycleCount.comments.label')}</th>
```

Comment:
we should use g:message

Reply:
Right... I keep forgetting about this one.